### ISSUE TITLE
A11y_.NETOnlineREPL_Output_AI4W : Elements that have scrollable content should be accessible by keyboard (.dotnet-online-output)

### ISSUE THREAD

**Issue by pranshu512** [dotnet/try#913]
“[Check out Accessibility Insights! ](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2;#BM_.NET Online REPL_Web_Feb2021;#.NET Online REPL;#Win10;#FTP;#DesktopWeb;#WCAG2.1.1;#Chrome;#Keyboard;#AI4W;#A11yAuto;

### **Environment Details:**
Application Name: .NET Online REPL
Operating System: Windows10
Browser Version : Google Chrome Version 87.0.4280.141 (Official Build) (64-bit)

### **Repro Steps:**

1. Open .NET Editor coding screen .
2. Tab to .NET Editor section and enter code which gives large output after running code such that vertical scroll bar appear on output section.
3. Run AI4W fast pass.

**Title:** 
WCAG 2.1.1: Elements that have scrollable content should be accessible by keyboard (.dotnet-online-output)

**Tags:**
Accessibility, WCAG 2.1.1, scrollable-region-focusable

**Issue:** 
Elements that have scrollable content should be accessible by keyboard (scrollable-region-focusable - https://accessibilityinsights.io/info-examples/web/scrollable-region-focusable)

**Target application:** 
Hello World - Introduction to C# interactive C# tutorial 

**Element path:** 
.dotnet-online-output

**Snippet:** 
<pre class="dotnet-online-output" aria-live="polite">Bill Bill Bill Bill Bill Bill Bill Bill Bill Bill Bill Bill Bill Bill Bill Bill Bill Bill Bill Bill Bill Bill Bill Bill Bill Bill Bill Bill Bill Bill Bill Bill Bill Bill Bill Bill Bill </pre>

**How to fix:** 
Fix any of the following:
  Element should have focusable content
  Element should be focusable

**Environment:** 
Chrome version 88.0.4324.150

====

This accessibility issue was found using Accessibility Insights for Web 2.23.0 (axe-core 4.1.1), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.

**Comment by fsteffi** [comment]
issue does not reproduce
![image](https://user-images.githubusercontent.com/24580570/210325839-7a048cf6-4266-4279-9421-14d9b6f388ce.png)

https://dotnet.microsoft.com/en-us/learn/dotnet/in-browser-tutorial/2

@LadyNaggaga 
could you close this bug

**Comment by Rama8345** [comment]
**Github Tags:**#Closed;
